nexus/pkg/server/handler.go

```diff
@@ -167,19 +179,33 @@ func (h *Handler) handleDefer(record *service.Record) {
 	request := record.GetRequest().GetDefer()
 	switch request.State {
 	case service.DeferRequest_BEGIN:
+		h.logger.Debug("handleDefer: begin", "stream_id", h.settings.RunId)
 	case service.DeferRequest_FLUSH_STATS:
+		h.logger.Debug("handleDefer: flush stats", "stream_id", h.settings.RunId)
```

Comment:
this is not needed we have a single logger top of swtich line 100
(same for all the rest)